code/datums/antagonists/datum_clockcult.dm

```diff
@@ -20,25 +20,28 @@
 				to_chat(AI, "<span class='userdanger'>Anomaly Detected. Returned to core!</span>") //The AI needs to be in its core to properly be converted
 		. = is_eligible_servant(new_owner.current)
 		if(!silent && new_owner.current)
-			if(issilicon(new_owner.current))
-				to_chat(new_owner.current, "<span class='heavy_brass'>You are unable to compute this truth! Your circuits surge with energy! Your world glows a brilliant yellow!\
-				<br>All at once it comes to you. Ratvar, the Clockwork Justiciar, lies in exile, derelict and forgotten in an unseen realm.</span>")
+			if(.)
+				to_chat(new_owner.current, "<span class='heavy_brass'>The world before you suddenly glows a brilliant yellow. [issilicon(new_owner.current) ? "You cannot compute this truth!" : \
+				"Your mind is racing!"] You hear the whooshing steam and cl[pick("ank", "ink", "unk", "ang")]ing cogs of a billion billion machines, and all at once it comes to you.<br>\
+				Ratvar, the Clockwork Justiciar, [GLOB.ratvar_awakens ? "has been freed from his eternal prison" : "lies in exile, derelict and forgotten in an unseen realm"].</span>")
+				flash_color(new_owner.current, flash_color = list("#BE8700", "#BE8700", "#BE8700", rgb(0,0,0)), flash_time = 50)
```

Comment:
Might wanna update the PR title to include that the messages are different now, too!